Google trends list index out of range error

I've run the bot on multiple devices and even my friend can confirm that this error just happens randomly. Usually the bot runs successfully but occasionally will error out. Here is the error:

```
2024-08-04 20:46:29,924 [INFO] [READ TO EARN] Completed the Read to Earn successfully !
2024-08-04 20:46:29,924 [DEBUG] googleTrendsShelf.__dict__ = {'dict': <dbm.dumb._Database object at 0x00000191F23D63D0>, '_protocol': 4, 'writeback': False, 'cache': {}, 'keyencoding': 'utf-8'}
2024-08-04 20:46:29,925 [DEBUG] google_trends = [('loadDate', datetime.date(2024, 8, 4))]
2024-08-04 20:46:29,925 [INFO] [BING] Starting Mobile Edge Bing searches...
2024-08-04 20:46:34,286 [INFO] [BING] 1/20
2024-08-04 20:46:34,615 [DEBUG] in __exit__ exc_type=<class 'IndexError'> exc_value=list index out of range traceback=<traceback object at 0x00000191F2848B00>
2024-08-04 20:46:34,715 [ERROR] 
Traceback (most recent call last):
  File "d:\MS-Rewards-Farmer\main.py", line 39, in main
    earned_points = executeBot(currentAccount, args)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\MS-Rewards-Farmer\main.py", line 266, in executeBot
    searches.bingSearches()
  File "d:\MS-Rewards-Farmer\src\searches.py", line 136, in bingSearches
    self.bingSearch()
  File "d:\MS-Rewards-Farmer\src\searches.py", line 147, in bingSearch
    rootTerm = list(self.googleTrendsShelf.keys())[1]
               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
2024-08-04 20:46:34,919 [INFO] [POINTS] Data saved for the next day.
```

--- a/src/searches.py
+++ b/src/searches.py
@@ -68,13 +68,13 @@ class Searches:
 
         if loadDate is None or loadDate < date.today():
             self.googleTrendsShelf.clear()
-            self.googleTrendsShelf[LOAD_DATE_KEY] = date.today()
             trends = self.getGoogleTrends(
                 browser.getRemainingSearches(desktopAndMobile=True).getTotal()
             )
             random.shuffle(trends)
             for trend in trends:
                 self.googleTrendsShelf[trend] = None
+            self.googleTrendsShelf[LOAD_DATE_KEY] = date.today()
             logging.debug(
                 f"google_trends after load = {list(self.googleTrendsShelf.items())}"
             )
@@ -144,7 +144,7 @@ class Searches:
         # Function to perform a single Bing search
         pointsBefore = self.browser.utils.getAccountPoints()
 
-        rootTerm = list(self.googleTrendsShelf.keys())[1]
+        rootTerm = list(self.googleTrendsShelf.keys())[0]
         terms = self.getRelatedTerms(rootTerm)
         logging.debug(f"terms={terms}")
         termsCycle: cycle[str] = cycle(terms)

